Update dotnet.md (#3119)
dotnet/EntityFramework.Docs: entity-framework/core/cli/dotnet.md

## `dotnet ef migrations remove`

Removes the last migration by calling the `Down` method of the latest migration. The command rolls back the code changes that were done for the latest migration.

Options:

| Option                 | Short             | Description                                                                     |
|:-----------------------|:------------------|:--------------------------------------------------------------------------------|

> | <nobr>`--force`</nobr> | <nobr>`-f`</nobr> | Revert the latest migration, that is, rolls back the changes that were applied to the database on the last migration. |

Reviewer: How about:

```suggestion
| <nobr>`--force`</nobr> | <nobr>`-f`</nobr> | Revert the latest migration, rolling back the changes that were applied to the database in the last migration. |
```
